src/actions/transactions.js

```diff
@@ -108,8 +108,9 @@ export const loadTransaction = ({ activePeer, id }) =>
     dispatch({ type: actionTypes.transactionCleared });
     getSingleTransaction({ activePeer, id })
       .then((response) => {
-        const added = (response.transaction.votes && response.transaction.votes.added) || [];
-        const deleted = (response.transaction.votes && response.transaction.votes.deleted) || [];
+        const transaction = response.data[0];
+        const added = (transaction.votes && transaction.votes.added) || [];
```

Comment:
I think the api changed, and now the object to be used is transaction.asset.votes, and requires to filter added & removed.
see
https://github.com/LiskHQ/lisk-hub/blob/99ae7afc92a7b796ef4b9f6299ba18a027277966/src/actions/transactions.js#L111